packages/did/README.md

```diff
@@ -25,19 +25,6 @@ const { register } = require('@tangleid/did');
 
 const seed = 'THISISTHESEEDOFTHETICACCOUNTANDISHOULDNOTGIVEITTOANYBODYELSE';
 const result = await register(seed, '0x2');
-/*
-{
-  did:
-    'did:tangleid:MoWYKbBfezWbsTkYAngUu523F8YQgHfARhWWsTFSN2U45eAMpsSx3DnrV4SyZHCFuyDqjvQdg7',
-  masterRoot:
-    'OMEDQKSVKAFHEX9OSTBZLFBGXNMSUO9LEUFFCDTOSIJXCQMNFIHDCKHMUJCYPXGXXMPLHIAMLXBFAZADA',
-  password: 'THISISTHESEEDOFTHETICACCOUNTANDISHOULDNOTGIVEITTOANYBODYELSE',
-  profileRoot:
-    'YMGVIKHFZIGUVANVHIUAKJUTPAZKBBLATTCOPJJAQPIKNUNYCKKHHHSUULSBYJVXDXDGUJKMGSIVZSPPI',
-  contactsRoot:
-    'PUZSASSTMBDGPPYJJEZZODAIFWBMIMPMBLWNMOHWVUQHVSAHEKO9LHUSFUZYJZ9DNKLU9MF9FOYXEWLTK'
-}
-*/
```

Comment:
I updated the README.md to reflect the code changes.
This comment should not exist in here, so I remove it.